redisinsight/api/package.json

```diff
@@ -87,6 +89,7 @@
     "node-version-compare": "^1.0.3",
     "quicktype-core": "^23.0.116",
     "redis": "^4.6.10",
+    "redis-parser": "3.0.0",
```

Comment:
why is this changed?

Reply:
Since we have patch for it we want to make sure that version is fixed